core/openapi/schemas/party.yaml

```diff
@@ -50,7 +50,7 @@ oneOf:
                 description: |-
                   The key name indicates the type of phone number (e.g. home,
                   work, fax, etc.). The value is the phone number itself.
-                pattern: "^+[1-9]{1}[0-9]{3,14}$"
+                # pattern: "^+[1-9]{1}[0-9]{3,14}$"
```

Comment:
@pvretano while this PR is closed in lieu of JSON-FG, the above (unrelated to `time`) fix was needed for (Python) JSONSchema validation tooling:

https://github.com/wmo-im/wcmp2/actions/runs/4277972994/jobs/7447254647#step:5:8

Snippet below:

```bash
for e in examples/*.json; do check-jsonschema --schemafile schemas/wcmp2-bundled.json $e; done
Error: schemafile was not valid: '^+[1-9]{1}[0-9]{3,14}$' is not a 'regex'
```

Wondering if there's any objection to relaxing / removing the regex.

Reply:
@mariadb-pvretano @pvretano yes.  Something like the below might work:

```
"pattern": "^\\\\s*(?:\\\\+?(\\\\d{1,3}))?[-. (]*(\\\\d{3})[-. )]*(\\\\d{3})[-. ]*(\\\\d{4})(?: *x(\\\\d+))?\\\\s*$"
```